lib/spack/spack/test/compilers/libraries.py

```diff
@@ -115,7 +118,7 @@ def test_implicit_rpaths(self, mock_gcc, dirs_with_libfiles, monkeypatch):
         retrieved_rpaths = detector.implicit_rpaths()
         assert set(retrieved_rpaths) == set(lib_to_dirs["libstdc++"] + lib_to_dirs["libgfortran"])
 
-    def test_compiler_environment(self, working_env, mock_gcc, monkeypatch):
+    def test_compiler_environment(self, mock_gcc, working_env, monkeypatch):
```

Comment:
Oops. As you can tell I tried something wrt `mock_gcc` but didn't put it back in the original location. Do you want me to fix that?

Reply:
Nvm